technical-writing-project-ideas.md

```diff
@@ -1,22 +1,57 @@
 
-4. Take part in refactoring the TiDB documentation for content reuse of Markdown files. Some existing documents that are a bit too long will be split into two or multiple separate documents, to make it easy for content reuse.
+- **Organization name**: PingCAP
 
-5. (Optional) Improve the [Chinese version of TiDB documentation](https://github.com/pingcap/docs-cn) to make it more user-friendly if the Technical Writer is good at both English and Chinese. This can include reorganizing the content in a single document or within several documents, to make it easier to use and understand.
+- **Organization description**: [PingCAP Inc.](https://pingcap.com/about/) is a software company founded in April 2015, dedicated to building [TiDB](https://github.com/pingcap/tidb), an open-source, distributed, Hybrid Transactional and Analytical Processing (HTAP) database. PingCAP is the primary developer, maintainer, and driver of TiDB and its associated [open-source communities](https://github.com/pingcap/community). It is a venture-backed company, which announced its 50 million USD Series C round financing in September 2018.
+
+- **Reference links:**
+
+    - open-source repository: <https://github.com/pingcap/tidb>
+    - Website: <https://pingcap.com>
+
+- **Email:** [gsod-tidb@pingcap.com](mailto:gsod-tidb@pingcap.com)
+
+- **Display name:**
+
+    - Administrator: [lilin90](https://github.com/lilin90)
+    - Mentor: Coco
+
+## Project ideas
+
+This section describes the project ideas of [TiDB documentation](https://pingcap.com/docs) in Google Season of Docs.
+
+### Project name
+
+TiDB 4.0 Documentation Improvement
+
+### Project description
+
+1. Improve the current [TiDB 4.0 documentation](https://pingcap.com/docs) to make it more user-friendly. This can include reorganizing the content in a single document or within several documents, to make it easier to use and understand.
+
+2. Execute user acceptance tests for TiDB 4.0 how-to guides and tutorials. This can include executing a usability test by following the existing document, such as testing the deployment of TiDB or TiDB ecosystem tools. In the process of testing, the technical writer might find it necessary to update some steps or outdated description or the content structure. Then they can file Pull Requests to improve the documentation based on tests.
+
+3. Review [GitHub Pull Requests about TiDB 4.0 documentation](https://github.com/pingcap/docs/pulls). This can include reviewing the language and content organization in docs Pull Requests filed by a contributor, a developer, a Database Administrator, or another Technical Writer.
+
+### Related material
+
+- The open-source project: <https://github.com/pingcap/tidb>
+- Existing documentation: <https://pingcap.com/docs>
+- [How-to guides and tutorials](https://pingcap.com/docs/stable/how-to/deploy/orchestrated/tiup/): topics about how to deploy, configure, scale, or monitor a TiDB cluster in different ways
```

Comment:
```suggestion
- [How-to guides and tutorials](https://pingcap.com/docs/v4.0/how-to/deploy/orchestrated/tiup/): topics about how to deploy, configure, scale, or monitor a TiDB cluster in different ways
```
https://pingcap.com/docs/v4.0/how-to/deploy/orchestrated/tiup/ does not include topics about how to configure and scale a TiDB cluster?

Reply:
Yes, but they can find those topics on the left navigation TOC.